src/tests/test_utils/common_test_utils/include/common_test_utils/test_common.hpp

```diff
@@ -18,7 +18,7 @@ class TestsCommon;
 
 // Temporal WA for applications.ai.vpu-accelerators.vpux-plugin repo
```

Comment:
Please update device name in comments from VPU to NPU.

Reply:
Done.